Dockerfile

```diff
@@ -9,20 +9,33 @@ LABEL org.label-schema.vendor="gyptazy"
 LABEL org.label-schema.url="https://proxlb.de"
 LABEL org.label-schema.vcs-url="https://github.com/gyptazy/ProxLB"
 
-# Install Python3
-RUN apk add --no-cache python3 py3-pip
+# --- Step 1 (root): system deps, user, dirs ---
+RUN apk add --no-cache python3 py3-pip py3-virtualenv \
```

Comment:
Starting with Py 3.3 we have venv support already in the base package. So, it should probably also work without that package.

Package version in Alpine: https://pkgs.alpinelinux.org/package/edge/main/x86/python3